packages/txservice/src/txservice.ts

```diff
@@ -152,7 +154,7 @@ export class TransactionService {
             { method, methodId, requestContext },
             `(${transaction.id}, ${transaction.attempt}) ${error}`,
           );
```

Comment:
if we want these (id especially) to be json-searchable, we should make sure they're logged in the json above under a common field. if this is within the `requestContext` already that's fine

Reply:
Will include that with these changes